source/rafcon/gui/helpers/state_machine.py

```diff
@@ -1394,6 +1396,32 @@ def ungroup_selected_state():
         return gui_helper_state.ungroup_state(selected_state_m)
 
 
+def change_background_color(state_model):
+    from rafcon.gui.utils.dialog import get_root_window
+    dialog = Gtk.ColorSelectionDialog('Change Background Color', parent=get_root_window())
+    default_button_response = -7
+    dialog.add_button('Set to Default', default_button_response)
+    response = dialog.run()
+    changed = False
+    if response == -5:
+        current_color = dialog.get_color_selection().get_current_color()
+        state_model.set_meta_data_editor('custom_background_color', True)
```

Comment:
Is it necessary to split the information about the background color into two properties? Why not just have one property `[custom_]background_color` that stores a custom color. If that property is not present, the default color is chosen.